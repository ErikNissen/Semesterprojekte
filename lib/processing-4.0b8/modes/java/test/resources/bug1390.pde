version https://git-lfs.github.com/spec/v1
oid sha256:1cf1af96aeccfbc5f4aae6032ba9cae4bf8497dc304cff058b077d1ea09e89bf
size 773
