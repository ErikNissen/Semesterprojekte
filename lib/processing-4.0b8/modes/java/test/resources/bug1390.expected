version https://git-lfs.github.com/spec/v1
oid sha256:37ddfa18f7e31571950ef22a7a9611768da7ad08ac92e0cc8100c7635f2ccc61
size 1425
